<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFAST</class>
 <widget class="QDialog" name="DialogFAST">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,0,0">
   <item row="1" column="2">
    <widget class="QSlider" name="sliderThreshold">
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QCheckBox" name="cBNMS">
     <property name="text">
      <string>non maxima suppression</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QCheckBox" name="cBDrawKeyPoints">
     <property name="text">
      <string>draw keypoints</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>threshold</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="cBxType">
     <item>
      <property name="text">
       <string>9_16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7_12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5_8</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" rowspan="6">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
